build/config.props

```diff
@@ -19,7 +19,7 @@
 
     <!-- ** Change for each new preview/rc -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.3</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">rc</ReleaseLabel>
 
     <IsEscrowMode>false</IsEscrowMode>
```

Comment:
Does IsEscrowMode need updated?

Reply:
Nope, that sounds great. 
Make sure you have an issue fro changing the IsEscrowMode property next week, otherwise we'd run into issues with opt prof.

Reply:
Maybe stupid question. When exactly we use IsEscrowMode?  I don't remember using it or updating it.